facia-json/src/main/scala/com/gu/facia/client/AmazonSdkS3Client.scala

```diff
@@ -1,18 +1,14 @@
 package com.gu.facia.client
 
-import com.amazonaws.regions.{Region, Regions}
-import com.amazonaws.services.s3.{AmazonS3, AmazonS3ClientBuilder}
+import com.amazonaws.regions.Regions
 import com.amazonaws.services.s3.model.AmazonS3Exception
+import com.amazonaws.services.s3.{AmazonS3, AmazonS3ClientBuilder}
 import org.apache.commons.io.IOUtils
+
 import scala.concurrent.{ExecutionContext, Future, blocking}
 import scala.util.Try
 
-/** For mocking in tests, but also to allow someone to define a properly asynchronous S3 client. (The one in the AWS
-  * SDK is unfortunately synchronous only.)
-  */
-trait S3Client {
-  def get(bucket: String, path: String): Future[FaciaResult]
-}
```

Comment:
Note the `com.gu.facia.client.S3Client` trait still exists, it's just moved to the `fapi-client-core` artifact - couldn't delete it without badly breaking binary compatability for existing users!